### PR TITLE
feat: add Mycoolplay

### PR DESCRIPTION
**Pull request by Theryx**
This pull request adds Mycoolplay to the catalogue
the app infos was copied from https://my-coolpay.com/en/

related to #1

### CODE REVIEW

File: src/res/apps.yaml [hunk 1]
```diff
@@ -32,22 +32,22 @@ items:
       - https://blog.osscameroon.com
       - https://miniyotas.osscameroon.com
     image: default.svg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:35]: It looks like your text editor, has reformatted the file, is it possible to change this line back, as they were before ?

File: src/res/apps.yaml [hunk 2]
```diff
@@ -32,22 +32,22 @@ items:
...
   - title: Python Cameroun
     category: community
     #max of five tags
     tags:
       - programming
       - developers
     description: |
-     The cameroon based python community organization.
+      The cameroon based python community organization.
```
Comment (elhmn) [inline src/res/apps.yaml:43]: ditto

File: src/res/apps.yaml [hunk 3]
```diff
@@ -32,22 +32,22 @@ items:
...
     website: https://pythoncm.org
     #twitter group handle
     twitter: https://twitter.com/PythonCameroun
     github_account: https://github.com/python-cameroun
     telegram: https://t.me/python_cameroon
     image: python_cameroun.jpeg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:50]: ditto

File: src/res/apps.yaml [hunk 4]
```diff
@@ -65,7 +65,7 @@ items:
     telegram: https://t.me/joinchat/FzwEmhVrJcOnxQdG6ETtCA
     slack: https://laravel.cm/join-slack
     image: laravel_cm.jpeg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:68]: ditto

File: src/res/apps.yaml [hunk 5]
```diff
@@ -79,7 +79,7 @@ items:
     playstore: https://play.google.com/store/apps/details?id=co.dklo.mbuntu
     appstore: https://about.dklo.co/mbuntu-beta-ios-en
     image: mbuntu.jpeg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:82]: ditto

File: src/res/apps.yaml [hunk 6]
```diff
@@ -91,21 +91,21 @@ items:
     website: https://www.aurionuniverse.com/mobile/aurionkgf
     playstore: https://play.google.com/store/apps/details?id=com.Kiroogames.AurionKGF
     image: aurionkgf.png
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:94]: ditto

File: src/res/apps.yaml [hunk 7]
```diff
@@ -91,21 +91,21 @@ items:
...
   - title: "Le Responsable Mboa"
     category: video game
     #max of five tags
     tags:
       - social
       - jeux videos
     description: |
-     Sous très haute instruction de Son Excellentissime Mougabiang, vous allez enfin pouvoir vivre une première semaine en tant que stagiaire dans le "Ministère Général", dans ce jeu de gestion/simulation de vie satyrique {ENCORE EN DEVELOPPEMENT}.
+      Sous très haute instruction de Son Excellentissime Mougabiang, vous allez enfin pouvoir vivre une première semaine en tant que stagiaire dans le "Ministère Général", dans ce jeu de gestion/simulation de vie satyrique {ENCORE EN DEVELOPPEMENT}.
```
Comment (elhmn) [inline src/res/apps.yaml:102]: ditto

File: src/res/apps.yaml [hunk 8]
```diff
@@ -91,21 +91,21 @@ items:
...
     website: https://www.leresponsable-mboa.com
     #twitter group handle
     twitter: https://twitter.com/ResponsableMboa
     playstore: https://play.google.com/store/apps/details?id=com.kiroogames.leresponsablemboa
     image: le_responsable_mboa.jpeg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:108]: ditto

File: src/res/apps.yaml [hunk 9]
```diff
@@ -118,7 +118,7 @@ items:
     twitter: https://twitter.com/afrotiles
     playstore: https://play.google.com/store/apps/details?id=com.elhmn.afrotiles
     image: afrotiles.jpeg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:121]: ditto

File: src/res/apps.yaml [hunk 10]
```diff
@@ -129,7 +129,7 @@ items:
     website: http://voyageurs.elhmn.com
     #twitter group handle
     image: voyageurs.png
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:132]: ditto

File: src/res/apps.yaml [hunk 11]
```diff
@@ -141,7 +141,7 @@ items:
     #twitter group handle
     twitter: https://twitter.com/fluttercm
     image: flutter_cm.jpeg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:144]: ditto

File: src/res/apps.yaml [hunk 12]
```diff
@@ -156,7 +156,7 @@ items:
     #github_account full url
     github_account: https://github.com/blurdylan/lamap-bot
     image: lamap_bot.png
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:159]: ditto

File: src/res/apps.yaml [hunk 13]
```diff
@@ -169,7 +169,7 @@ items:
     #twitter group handle
     twitter: https://twitter.com/Qui_Go
     image: quigo.png
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:172]: ditto

File: src/res/apps.yaml [hunk 14]
```diff
@@ -182,15 +182,33 @@ items:
     twitter: https://twitter.com/MyStoreAfrica
     telegram: https://t.me/mystoreafrica_bot
     image: https://mystore.africa/assets/img/logo-two.png
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:185]: ditto

File: src/res/apps.yaml [hunk 15]
```diff
@@ -182,15 +182,33 @@ items:
...
     #max of five tags
     tags:
       - hosting
     description: |
-      Mettez vos sites web en ligne facilement avec Hostme.space. 
+      Mettez vos sites web en ligne facilement avec Hostme.space.
     website: https://www.hostme.space
     telegram: https://t.me/hostme_space
     image: https://hostme.space/front_asset/img/logo/logo.png
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:196]: This line should not be indented

File: src/res/apps.yaml [hunk 16]
```diff
@@ -182,15 +182,33 @@ items:
...
+      - api
+      - programming
+      - developers
+    description: |
+      My-CoolPay aggregates all digital payment methods to enable businesses and individuals to raise funds online.
+    website: https://my-coolpay.com/fr/
+    #twitter group handle
+    twitter: https://twitter.com/MyCoolPay
+    #dikalo group handle or link
+    dikalo:
```
Comment (elhmn) [inline src/res/apps.yaml:210]: since there is no `dikalo` link maybe it is worth removing it ?

**Review by elhmn — COMMENTED** [review]
Thanks for submitting this issue 🙇‍♂️ , I just left few comments regarding the file format that was changed and a `dikalo` field that we could remove. 👍

File: src/res/apps.yaml (hunk 1 above)
Comment (Theryx) [reply src/res/apps.yaml:35]: Alright, thanks for pointing that. I did not notice. please help me interpret what changed?
what does the blue and red represent?

Comment (elhmn) [reply src/res/apps.yaml:35]: Sure, The green line refers to what you have added in your PR, the stronger green on line `35` highlight a space between the beginning of the line and the comment. You should remove this space, I guess

**Comment by elhmn** [comment]
I am closing this PR as a fix PR was created here https://github.com/osscameroon/camerapps.com/pull/10

**Pull request closed without merging**